### DISCUSSION TITLE
Plotting with 2 temp SSD ?

### DISCUSSION THREAD

**Discussion by louisaiva** [Chia-Network/chia-blockchain#5534] [Support]
Hello !
I am trying to create plots on my 2 TB HDD, with one 256 GB SSD and a 512 GB SSD. 
my first SSD is in reality 238 GB (D:/) and my second is almost full with 50 GB left.
I created a new partition E:/ for the 50 GB left on my 512 GB SSD

As I need ~256 GB temp space to create a 108 GB plot, I tried to used D:/ and E:/ as temp space.
But the plotting is stucked at 32% with Error 1. In fact, it filled entirely D:/ but didn't use E:/ at all...
`Only wrote 0 of 1048570 bytes at offset 332396690 to "D:" ... . Error 1. Retrying in five minutes.`

Is here any way for me to use 2 temp space to create plots ? It's like it does not work..
Maybe by using the cli chia ?

Thanks for help.
I am using Chia Gui 1.1.5

**Comment by jefersonrosa10** [comment]
you will enter the folder where you are plotting on the hd with the largest space, you will copy plot.table1.tmp of 15 gb and plot.table2.tmp of 21 gb, to the hd with 50 gb of space, after you finish copying the file you will delete the file you are plotting from, when plotting an error message from the plots that you delete right in front and just copy it again from the plot folder

**Reply by louisaiva** [reply]
ok I will try that !

**Comment by jefersonrosa10** [comment]
you will enter the folder where you are plotting on the hd with the largest space, you will copy plot.table1.tmp of 15 gb and plot.table2.tmp of 21 gb, to the hd with 50 gb of space, after you finish copying the file you will delete the file you are plotting from, when plotting an error message from the plots that you delete right in front and just copy it again to the plotting folder

**Comment by knightblackm** [comment]
i saw a video on youtube, youtuber extended two ssd (same size, same model and same brand) together. you can try that
and make a 768GB.

**Reply by knightblackm** [reply]
or use copy some file at end of phase two!

**Comment by BrandtH22** [comment]
@louisaiva , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
